core/src/test/kotlin/org/stellar/anchor/auth/JwtServiceTest.kt

```diff
@@ -37,13 +34,20 @@ internal class JwtServiceTest {
   fun setup() {
     secretConfig = mockk()
     custodySecretConfig = mockk()
-    every { secretConfig.sep6MoreInfoUrlJwtSecret } returns "jwt_secret"
-    every { secretConfig.sep10JwtSecretKey } returns "jwt_secret"
-    every { secretConfig.sep24InteractiveUrlJwtSecret } returns "jwt_secret"
-    every { secretConfig.sep24MoreInfoUrlJwtSecret } returns "jwt_secret"
-    every { secretConfig.callbackAuthSecret } returns "jwt_secret"
-    every { secretConfig.platformAuthSecret } returns "jwt_secret"
-    every { custodySecretConfig.custodyAuthSecret } returns "jwt_secret"
+    every { secretConfig.sep6MoreInfoUrlJwtSecret } returns
+      "jwt_secrejwt_secrejwt_secrejwt_secrettttjwt_secrejwt_secrejwt_secrejwt_secrejwt_secrettttjwt_secrettjwt_secrejwt_secrejwt_secrejwt_secrettttjwt_secret".toSecretKey()
```

Comment:
nit: we can move this into a test constants file so we don't have to copy these long strings everywhere.